frontend/src/scenes/me/Settings/UserDetails.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Input } from 'antd'
+import { useValues } from 'kea'
+import { userLogic } from 'scenes/userLogic'
+import { LemonButton } from 'lib/components/LemonButton'
+import { Field } from 'lib/forms/Field'
+import { VerticalForm } from 'lib/forms/VerticalForm'
+
+export function UserDetails(): JSX.Element {
+    const { userLoading, isUserDetailsSubmitting } = useValues(userLogic)
+
+    return (
+        <VerticalForm
+            logic={userLogic}
+            formKey="userDetails"
+            enableFormOnSubmit
+            style={{
+                maxWidth: 400,
+            }}
+        >
+            <Field name="first_name" label="Name">
+                <Input
+                    className="ph-ignore-input"
+                    autoFocus
+                    data-attr="settings-update-first-name"
+                    placeholder="Jane Doe"
+                    disabled={userLoading}
+                />
+            </Field>
+
+            <LemonButton
+                type="primary"
+                htmlType="submit"
+                loading={isUserDetailsSubmitting}
+                data-attr="user-details-submit-bottom"
+            >
+                Update Details
```

Comment:
"Details" is a very generic term, but at the same time there only is one "detail": the name. Which is a bit of a mismatch. Maybe we should just do "Save Name" for now?
As a side note, we generally go for Sentence case in buttons now, in which case it'd be "Save name", but that'd also require updating other buttons on the page.

Reply:
Thats a fair point. 

Updating the other buttons is fine as I swapped them to LemonButtons anways

Reply:
As a temporary solution I removed the "details" title so we just have the subprop name and a save button